src/main/java/de/tum/cit/aet/artemis/core/security/jwt/JWTFilter.java

```diff
@@ -9,6 +9,7 @@
 import jakarta.servlet.http.Cookie;
 import jakarta.servlet.http.HttpServletRequest;
 
+import org.springframework.lang.Nullable;
```

Comment:
There's an architecture violation here. Can you update the import to be `import jakarta.annotation.Nullable;`?

```
ArchitectureTest > testNullnessAnnotations() FAILED
    java.lang.AssertionError: Architecture Violation [Priority: MEDIUM] - Rule 'no code units should be annotated with not Annotation in package jakarta.annotation and Annotation with simple name Nullable' was violated (3 times):
    Method <de.tum.cit.aet.artemis.core.security.jwt.JWTFilter.extractValidJwt(jakarta.servlet.http.HttpServletRequest, de.tum.cit.aet.artemis.core.security.jwt.TokenProvider)> is annotated with not Annotation in package jakarta.annotation and Annotation with simple name Nullable in (JWTFilter.java:55)
    Method <de.tum.cit.aet.artemis.core.security.jwt.JWTFilter.getJwtFromBearer(java.lang.String)> is annotated with not Annotation in package jakarta.annotation and Annotation with simple name Nullable in (JWTFilter.java:86)
    Method <de.tum.cit.aet.artemis.core.security.jwt.JWTFilter.getJwtFromCookie(jakarta.servlet.http.Cookie)> is annotated with not Annotation in package jakarta.annotation and Annotation with simple name Nullable in (JWTFilter.java:73)
```

Reply:
ah thanks i will